Server/Resources.xcassets/alerts.dataset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  },
+  "data" : [
+    {
+      "idiom" : "universal",
+      "filename" : "alerts.json"
```

Comment:
Could you please rename?
`alerts.json` -> `springboard-alerts.json`
`alerts.dataset` -> `springboard-alerts.dataset`